build.sh

```diff
@@ -33,6 +33,12 @@ docker stop $dockerdname
 docker rm $dockerdname
 
 # Now build and copy out the acap
+
+# update the manifest.json file to list the correct architecture
```

Comment:
The architecture gets injected in to the manifest.json file at build time.
Does this not work for this application?